docs/References/Runme_Parameters.md

```diff
@@ -0,0 +1,154 @@
+## `freemocap.RunMe()` Specify recording session  paramters 
+
+The `freemocap.RunMe()` function takes a number of parameters that can be used to alter it's default behavior in important ways. Here are the default parameters along with a followed by a brief description of each one. 
+
+---
+### RunMe - Default parameters
+```python
+#in `freemocap/fmc_runme.py`
+def RunMe(sessionID=None,
+        stage=1,
+        useOpenPose=False, 
+        runOpenPose = True, 
+        useMediaPipe=True,
+        runMediaPipe=True,
+        useDLC=False,
+        dlcConfigPath=None,
+        debug=False,
+        setDataPath = False,
+        userDataPath = None,
+        recordVid = True,
+        showAnimation = True,
+        reconstructionConfidenceThreshold = .7,
+        charucoSquareSize = 36, #mm
+        calVideoFrameLength = .5,
+        startFrame = 0,
+        useBlender = False,
+        resetBlenderExe = False,
+      	get_synced_unix_timestamps = True,
+        good_clean_frame_number = 0,
+        bundle_adjust_3d_points=False
+        ):
+```
+
+### [RunMe input parameters](#runme-input-parameters)
+- `sessionID`
+  - Type - (str) 
+  - [Default] - None.
+  - Indentifying string to use for this session. 
+  - If creating a new session, default behavior is to autogerate SessionID is based on date and time that the session was recorded
+  - If re-processing a previously recorded session, this value specifies which session to reprocess (must be the name of a folder within the `FreeMoCap_Data` folder)
+  - 
+- `stage`
+  - [Type] - Int 
+  - [Default] - 1
+  - Which processing stage to start from. Processing stages are deined in more  detail in [#processing-stages](#processing-stages) 
+  
+  ```
+  stage 1 - Record Raw Videos
+  stage 2 - Synchronize Videos
+  stage 3 - Camera Calibration
+  stage 4 - 2d Tracking and 3d Calibration
+  stage 5 - Create output files (using Blender)
+  stage 6 - Create output animation (Matplotlib)
+  ```  
+- `useMediaPipe`
```

Comment:
we might want to clarify a bit more the difference between the `use` and `run` flags for all the tracking methods. I don't think its intuitive from the flag names